qemu/tests/qemu_guest_agent.py

```diff
@@ -130,19 +163,18 @@ def gagent_start(self, params, vm, *args):
         if not gagent_start_cmd:
             return
 
-        error.context("Try to start 'qemu-guest-agent'.", logging.info)
-        session = self._get_session(params, vm)
+        error_context.context("Try to start 'qemu-guest-agent'.", logging.info)
         s, o = self._session_cmd_close(session, gagent_start_cmd)
```

Comment:
delete  _session_cmd_close() function.  Checked there is no other place to use it.
using session.cmd_status_output() instead of it.
close session in setup().